File: test6.py
numBit = 0
num = eval(input())

for i in range(2, -1, -1):
  numBit = num // pow(10, i)
  if(i == 2):
    for j in range(numBit):
      print("B", end = "")
  elif(i == 1):
    for j in range(numBit):
      print("S", end = "")
  elif(i == 0):
    j = 1
    while(j < numBit + 1):
      print(j, end = "")
      j += 1
  num = num - numBit * pow(10, i)

#print()

File: test6.cpp
#include<iostream>
#include<string>
#include<math.h>

int main()
{
  int num, numBit;

  std::cin >> num;

  for(double i = 2; i >= 0; i--)
  {
    numBit = num / pow(10, i);
    switch((int)i)
    {
      case 2:
        for(int j = 0; j < numBit; j++)
          std::cout << "B";
        break;
      case 1:
        for(int j = 0; j < numBit; j++)
          std::cout << "S";
        break;
      case 0:
        for(int j = 1; j < numBit + 1; j++)
          std::cout << j;
        break;
    }
    num = num - numBit * pow(10, i);
  }

  std::cout << std::endl;
  system("pause");
}